pkg/cmd/workspace/code.go

```diff
@@ -111,6 +109,16 @@ var CodeCmd = &cobra.Command{
 			}
 		}
 
+		project := workspace.GetProjectByName(projectName)
+		if project != nil {
+			if project.State != nil && (project.State.Uptime == nil || *project.State.Uptime == 0) {
+				views.RenderInfoMessage(fmt.Sprintf("project '%s' from workspace '%s' is not in running state", projectName, *workspace.Name))
```

Comment:
`project` => `Project`

Reply:
shouldn't we not name variables by starting with a lowercase though ?

Reply:
@titanventura I didn't mean the variable but the string in `fmt.Sprintf`

Reply:
hey @Tpuljak , these comments are resolved, please do take a look. Thanks !